numpy/f2py/crackfortran.py

```diff
@@ -2551,10 +2551,8 @@ def get_parameters(vars, global_params={}):
                 v = ''.join(tt)
 
             elif iscomplex(vars[n]):
-                # FIXME complex numbers may also have exponents
-                if v[0] == '(' and v[-1] == ')':
-                    # FIXME, unused l looks like potential bug
-                    l = markoutercomma(v[1:-1]).split('@,@')
+                outmess(f'get_parameters[TODO]:'
```

Comment:
```suggestion
                outmess(f'get_parameters[TODO]: '
```
Else this will print `TODO]:implement` with no space